lib/setup.js

```diff
@@ -100,7 +101,7 @@ function setup_test_instance(opt, cb) {
             debug('bouncer active on port %d', app_port);
 
             if (!config.tunnel) {
-                return cb(null, 'http://localhost:' + app_port + '/__zuul');
+                return cb(null, 'http://' + loopback + ':' + app_port + '/__zuul');
             }
```

Comment:
This looks like it's waking up `zalgo`.

Reply:
Good catch.

Reply:
Actually zalgo stays asleep, the function that contains this line is an event handler that already fires asynchronously.